<?xml version="1.0"?>
<launch>
	
	<remap from="ar_pose_marker" to="ar_blocks_environment/ar_pose_marker" />
	
	<include file="$(find ar_track_alvar)/launch/baxter_bundle_no_xtion.launch" />	
	
	<rosparam ns="ar_blocks_environment" command="load" file="$(find moveit_simple_grasps)/config/baxter_grasp_data.yaml" />
  <node name="ar_blocks_interface" pkg="ar_blocks" type="ar_blocks_interface" respawn="true" output="screen" />
	<node name="ar_blocks_environment" pkg="ar_blocks" type="ar_blocks_environment" respawn="true" output="screen" />
	
</launch>
